internal/appsec/dyngo/instrumentation/httpsec/http.go

```diff
@@ -162,7 +168,10 @@ func fromContext(ctx context.Context) *Operation {
 // finish event up in the operation stack.
 func (op *Operation) Finish(res HandlerOperationRes) json.RawMessage {
 	dyngo.FinishOperation(op, res)
-	return op.events
+	if events := op.Events(); len(events) > 0 {
+		return events[0]
+	}
+	return json.RawMessage{}
```

Comment:
I think we should now change return type to `[]json.RawMessage` and simply return `op.Events()`.
The fact this `events[0]` relies on the fact the HTTP WAF is called once in a hidden detail now, that was previously enforced by the now removed `AddSecurtyEvent()` method.

This change would impact other helper functions of `httpsec`.